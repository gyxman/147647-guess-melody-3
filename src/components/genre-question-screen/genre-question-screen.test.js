import React from "react";
import renderer from "react-test-renderer";
import GenreQuestionScreen from "./genre-question-screen.jsx";

it(`Если приложение загрузилось, то компонент GenreQuestionScreen отрисовался`, () => {
  const tree = renderer
    .create(<GenreQuestionScreen
      question={{answers: [], genre: ``}}
      onAnswer={() => {}}
      renderPlayer={() => {}}
      onChange={() => {}}
      userAnswers={[false, false, false, false]}
    />, {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
