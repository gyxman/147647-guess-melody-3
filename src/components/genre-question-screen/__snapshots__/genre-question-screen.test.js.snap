// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Если приложение загрузилось, то компонент GenreQuestionScreen отрисовался 1`] = `
<section
  className="game__screen"
>
  <h2
    className="game__title"
  >
    Выберите 
    
     треки
  </h2>
  <form
    className="game__tracks"
    onSubmit={[Function]}
  >
    <button
      className="game__submit button"
      type="submit"
    >
      Ответить
    </button>
  </form>
</section>
`;
